models/helper/MarginHelper.py

```diff
@@ -1,6 +1,6 @@
 def calculate_margin(buy_size: float = 0.0, buy_filled: int = 0.0, buy_price: int = 0.0, buy_fee: float = 0.0,
                      sell_percent: float = 100, sell_price: float = 0.0, sell_fee: float = 0.0,
-                     sell_taker_fee: float = 0.0, debug: bool = False, exchange: str = 'coinbasepro') -> float:
+                     sell_taker_fee: float = 0.0, debug: bool = False, exchange: str = 'coinbasepro', simulation: bool = False) -> float:
```

Comment:
not related to that PR,
could you change the return type from `float` to `[float, float, float]`?

Reply:
Done!